clean_enoa/call_inegi.py

```diff
@@ -0,0 +1,60 @@
+import requests
+import os
+import json
+
+
+def output(content, index):
+    per = content["Series"][0]["OBSERVATIONS"][index]["TIME_PERIOD"]
+    val = content["Series"][0]["OBSERVATIONS"][index]["OBS_VALUE"]
+    print(per, val)
+
+
+def get_content(url):
+    response = requests.get(url)
+    if response.status_code == 200:
```

Comment:
¿Por favor podrías encapsular el número mágico?

```suggestion
    ok_status = 200
    if response.status_code == ok_status:
```

Reply:
Primero había tomado tu sugerencia al pie de la letra, pero ahora me pareció suficiente con este cambio:
```python
is_status_ok = response.status_code == 200
if is_status_ok:
```
¿Te sigue pareciendo un número mágico?